delegates.txt

```diff
@@ -425,6 +425,7 @@ Ron Buckton (RBN)
 Rongjian Zhang (ZRJ)
 Ross Kirsling (RKG)
 Ruben Bridgewater (RBR)
+Ryan Cavanaugh (RJC)
```

Comment:
```suggestion
Ryan Cavanaugh (RCH)
```

Our convention is first1-first2-last which helps out notetakers.  It's not mandatory.  Would you consider the change?

Reply:
ping @RyanCavanaugh

Reply:
Sure